sdk/mediaservices/test-resources.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
@benbp if we end up needing a separate test-resources for mgmt and data do we have a good way to split these files?

Reply:
Not at the moment since it's still based on the service directory, but we could add a parameter to the live test yaml to support different paths for the deployment template file.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/2609